docs/new-architecture-backward-turbomodule.md

```diff
@@ -0,0 +1,709 @@
+---
+id: new-architecture-backward-compatible-turbomodule
+title: Create a Backward Compatible Turbomodule
+---
+
+<NewArchitectureWarning/>
+
+This guide describes how to create a Turbomodule, by keeping it backward compatibile with the old Architecture. To achieve this result, we first need to create a Native Module and then to build a Turbomodule on top of it.
+
+The guide is organized as it follows:
+
+1. [What is a Turbomodule, In Short](#what-is-a-turbomodule-in-short)
+1. [Create the Native Module](#create-the-native-module)
+   1. [Creating the Structure](#creating-the-structure)
+   1. [Create a Testing Environment](#create-a-testing-environment)
+   1. [Define the JS Shared Code](#define-the-js-shared-code)
+   1. [Implement the iOS Code](#implement-the-ios-code)
+   1. [Implement the Android Code](#implement-the-android-code)
+   1. [Test the Native Module](#test-the-native-module)
+1. [Create the Turbomodule](#create-the-turbomodule)
+   1. [Create the New Architecture Testing Environment](#create-the-new-architecture-testing-environment)
+   1. [Adding the JS Spec](#adding-the-js-spec)
+   1. [Generate the Code](#generate-the-code)
+   1. [Update the iOS Code](#update-the-ios-code)
+   1. [Update the Android Code](#update-the-android-code)
+   1. [Test the Turbomodule](#test-the-turbomodule)
+1. [Summary](#summary)
```

Comment:
```suggestion
The guide is organized as it follows:

1. [What is a Turbomodule, In Short](#what-is-a-turbomodule-in-short)
1. [Create the Native Module](#create-the-native-module)
   1. [Creating the Structure](#creating-the-structure)
   1. [Create a Testing Environment](#create-a-testing-environment)
   1. [Define the JS Shared Code](#define-the-js-shared-code)
   1. [Implement the iOS Code](#implement-the-ios-code)
   1. [Implement the Android Code](#implement-the-android-code)
   1. [Test the Native Module](#test-the-native-module)
1. [Create the Turbomodule](#create-the-turbomodule)
   1. [Create the New Architecture Testing Environment](#create-the-new-architecture-testing-environment)
   1. [Adding the JS Spec](#adding-the-js-spec)
   1. [Generate the Code](#generate-the-code)
   1. [Update the iOS Code](#update-the-ios-code)
   1. [Update the Android Code](#update-the-android-code)
   1. [Test the Turbomodule](#test-the-turbomodule)
1. [Summary](#summary)
```

Can we remove the in-content ToC? We used those in the past, but since migration to Docusaurus V2, the ToC is generated automatically, so we stick to that. Also maintain manually the links in there, in a long term usually leads to the broken links anyway after few iteration of changes.

<img width="1098" alt="Screenshot 2022-03-21 123443" src="https://user-images.githubusercontent.com/719641/159253872-24a3d7c9-cf30-4820-b03a-c894cfcd29de.png">